params.go

```diff
@@ -181,6 +181,12 @@ func IntToPointer(i int) *int {
 	return &i
 }
 
+// StringToPointer is a function for converting strings into string pointers.
+// This is useful when passing in options to operations.
+func StringToPointer(s string) *string {
```

Comment:
It's different because it's called `StringToPointer`. 

Yes, good catch. IIRC, I saw `MaybeString` and without reading it, disregarded it as something that would check if something was a string, bool, int, etc.